### DISCUSSION TITLE
How can i have 2 ranks in the townychat?

### DISCUSSION THREAD

**Discussion by TheWeirdCoder** [TownyAdvanced/Towny#6424] [Q&A] [answered]
So i want it to say [Mod] [Saint] if the staff member bought a rank so how can i have 2 ranks in the towny chat? [Both with colors]

heres my current towny chat config!
global: '{townytagoverride}{townycolor}{permprefix}{modplayername}{townypostfix}{permsuffix}&f:{msgcolour} {msg}'

**Comment by LlmDl** [comment, marked as answer]
I'm not sure how you have your luckperms configured, you might be able to do it with {permuserprefix} and {permgroupprefix} but you're probably best off using LP's PAPI placeholders.

**Reply by TheWeirdCoder** [reply]
Noted thanks!
